demo/index.html

```diff
@@ -38,6 +38,7 @@
   selector.addEventListener('change', function() {
     document.documentElement.lang = this.value;
     I18nMsg.lang = this.value;
+    Platform.performMicrotaskCheckpoint();
```

Comment:
Ah actually, this should be unnecessary. Polymer no longer uses `object.observe()`.  `performMicrotaskCheckpoint` was to flush the data system, where the polyfill was used and o.o() wasn't available.

I haven't ran your demo. Is there actually a problem in polyfill'd browsers?